include/multipass/utils.h

```diff
@@ -235,6 +235,8 @@ class Utils : public Singleton<Utils>
     virtual QString make_uuid(const std::optional<std::string>& seed = std::nullopt) const;
     virtual void sleep_for(const std::chrono::milliseconds& ms) const;
     virtual bool is_ipv4_valid(const std::string& ipv4) const;
+
+    virtual Path default_mount_target_from_source(const Path& source) const;
```

Comment:
Could we have a shorter name, please? C++ is verbose enough as it is, and source is already mentioned in the parameter, so perhaps just:

```suggestion
    virtual Path default_mount_target(const Path& source) const;
```

Reply:
Sure, makes sense. I'll change that everywhere.